README.md

```diff
@@ -7,9 +7,61 @@
 An [Apache 2.0](https://github.com/allenai/allennlp/blob/master/LICENSE) NLP research library, built on PyTorch,
 for developing state-of-the-art deep learning models on a wide variety of linguistic tasks.
 
-## Quickstart
+## Running AllenNLP
+
+The preferred way to install AllenNLP is via `pip`.
+
+### Setting up a virtual environment
+
+[Conda](https://conda.io/) can be used set up a virtual environment
+with the version of Python required for AllenNLP and in which you can
+sandbox its dependencies:
```

Comment:
I'd give a full stop here, and add a sentence saying something like "if you already have a python 3.6 environment you want to use, you can skip to the 'installing via pip' section."

Reply:
I would promote that users set up a virtual environment because it's something we can (almost) guarantee will work.  I think we should strive to have very clear instructions for new users who are potentially unfamiliar with the Python ecosystem.  Advanced users can probably figure out what they do from the detailed instructions, but beginners will trip up on too many options.